### ISSUE TITLE
Perfect Competition - Course Notes

### ISSUE THREAD

**Issue by ttttonyhe** [HelipengTony/devDiary#76]
## Definition

Perfect competition is a market in which:

1. Many firms sell identical products to many buyers
2. There are no restrictions to entry into the industry
3. Established firms have no advantages over new ones
4. Sellers and buyers are well informed about prices

<br/>

## How Perfect Competiton arises

When the firm's minimum efficient scale is samll relative to market deman, so there is room for many firms in the market

<br/>

## Price Takers

In perfect competitive market, a firm cannot influence the market price, so a firm sets its own price equal to the market price.

Each firm produces a small amount of the total output of the the good or supply, so increasing or decreasing a firm's output will **have no influence on the total supply** or the price of the good or service.

Each firm's output is a **perfect substitute** for the output of other firms, so the demand for each firm is perfect elastic.

A firm could **lower its price** but that would be equivalent to taking an economic loss because it can sell all they want at the going market price. At the lower price the firm will have **zero economic profit**.

<br/>

## Economic profit and revenue

To achieve maximum economic profit, a firm must decide:

1. how to produce at the minimum cost
2. how much to produce
3. whether to enter or exit the market.

In a perfect competitive market, MR is constant and equal to the market price.

<br/>

## The firm's output decision

![](https://static.ouorz.com/fromMac/perfect_competition_firms_output_decision.png)

To maximize economic profit, a firm would choose to produce 9 units of good as TR - TC is maximized.

<br/>

## Marginal analysis

MC < MR: economic profit increases if output increases

MC = MR: economic profit decrease if output changes, economic profit is maximized

MC > MR: economic profit decrease if output increases

<br/>

## Temporary shutdown

If a firm is experiencing an ecnomic loss, it must decide whether to exit or to stay in the market.

If a firm choose to stay in the market, it must decide whether to produce something or shutdown temporarily.

A firm may choose to stop its production if the firm believes the loss is short-lived or temporary in nature.

<br/>

## A firm's loss

Economic loss = $TFC + TVC - TR = TFC + (AVC - P) \times Q$

<br/>

## The shutdown point

At the shutdown point:

1. A firm is indifferent between producing and shutting down temporarily
2. The firm incurs a loss equal to total fixed cost ($TFC$)

A firm shutdown when the price or maginal revenue curve is below the marginal varibale cost curve

<br/>

## Output, Price, and profit in the short-run

Short-run Equilibrium

![](https://static.ouorz.com/fromMac/perfect_competition_short_run_equilibrium.png)

At the shutdown price ($17), some firms will produce the shutdown quantity (7 sweaters) and others will produce zero.

<br/>

## Short-run Profits and Losses

![](https://static.ouorz.com/fromMac/perfect_competition_short_run_profits_and_loss.png)

When a firm incurs an ecomonic loss (MR < TC), it does not necessarily mean that the firm will want to shutdown right away.

In the long-run, all firms will be making normal profits (breaks even) as other firms can enter or exit the market.

<br/>

## Breaking even in the long-run

When firms in a competitive market make an economic profit, it will attract other firms to enter the market, which causes a rightward shift of the industry supply curve. As supply increases, the price falls so does the economic profit. The process continues until no economic profit results.

Generally, the price will be equal to minimum $ATC$ in the long-run,

firms make zero economic profit and have no incentive to exit the market.

<br/>

## Technological Advances Changes Supply

When a new technology becomes available that lowers production costs, the first firms to use it make economic profit. But as more firms begin to use the new technology, market supply increases and the price falls.

![](https://static.ouorz.com/fromMac/perfect_competition_short-run_tech_advances_changes_supply.png)

<br/>

## Efficient use of Resources

1. Each firm produces at the lowest possible average total cost (minimum point of the long-run average cost curve). Consumers pay the lowest possible price that keeps firms in business $P = minimum ATC$
2. Each firm uses the least-cost technology. $P = MR = MC$
3. In the long-run equilibrium total surplus is maximized
